src/components/views/EResource.js

```diff
@@ -79,7 +80,7 @@ const EResource = ({
                       onClick={handlers.onToggleTags}
                     />
                   }
-                  {eresource.class === resourceClasses.PCI &&
+                  { (eresource.class === resourceClasses.PCI || (eresource.class === resourceClasses.TITLEINSTANCE && handlers.isSuppressFromDiscoveryEnabled('title'))) &&
```

Comment:
Re-thinking this, I dont think we should hide the edit button here? As we only have that behavior for a user with no permission to edit. At best, we should probably disable the button with a tooltip explaing the reason or just give them access to the edit pane and the option to close that form (default behavior currently) as none of the fields would be editable in this case.

Reply:
so what behaviour should I implement? Could we clarify this in the meeting today?